src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -84,6 +84,7 @@
 #include "transformations/op_conversions/convert_pad_to_group_conv.hpp"
 #include "transformations/op_conversions/convert_prior_box_v8_to_v0.hpp"
 #include "transformations/op_conversions/convert_reduce_to_pooling.hpp"
+#include "transformations/op_conversions/convert_reduce_to_reshape.hpp"
```

Comment:
If you are going to add a new pass to common passes, you should validate for CPU too at least. 
Or you can apply it only to the gpu pass?

Reply:
Oh I didn't catch this. Let's add it to gpu pass only.

Reply:
Thanks i will removed.